tables/vaccines.csv

```diff
@@ -32,4 +32,9 @@ Sputnik Light + Sputnik V,Viral vector,Gamaleya Research Institute of Epidemiolo
 BBIBP-CorV + ZF2001,Inactivated virus,Sinopharm Beijing,China,3,31
 Ad26.COV2.S + mRNA-1273,Mixed,,,2,32
 Nanocovax,Protein subunit,Nanogen Pharmaceutical Biotechnology JSC,Vietnam,2,33
-BNT162b2 + CoronaVac,Mixed,,,2,34
\ No newline at end of file
+BNT162b2 + CoronaVac,Mixed,,,2,34
+mRNA-LNP-WT,mRNA,Taiwan Institute of Biomedical Sciences,Taiwan,2,35
+mRNA-LNP-Delta,mRNA,Taiwan Institute of Biomedical Sciences,Taiwan,2,36
+mRNA-LNP-Omicron,mRNA,Taiwan Institute of Biomedical Sciences,Taiwan,2,37
+mRNA-LNP-Hybrid,mRNA,Taiwan Institute of Biomedical Sciences,Taiwan,2,38
```

Comment:
@zachandfox, Please change _mRNA-LNP-Hybrid_ to _mRNA-LNP-Omicron-Delta-RBD_. 
Please move all the files related to this study into *excluded* folder, we may need the data if the vaccine is in clinical trials or is authorized in the future.
Please also remove the vaccine name from the vaccines table.